schema/manifest_backwards_compatibility_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema_test
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/opencontainers/image-spec/schema"
+)
+
+var compatMap = map[string]string{
+	"application/vnd.docker.distribution.manifest.list.v2+json": "application/vnd.oci.image.manifest.list.v1+json",
+	"application/vnd.docker.distribution.manifest.v2+json":      "application/vnd.oci.image.manifest.v1+json",
+	"application/vnd.docker.image.rootfs.diff.tar.gzip":         "application/vnd.oci.image.rootfs.tar.gzip",
+	"application/vnd.docker.container.image.v1+json":            "application/vnd.oci.image.serialization.config.v1+json",
```

Comment:
lgtm

On 16 June 2016 at 19:21, Brandon Philips <notifications@github.com> wrote:

> In schema/manifest_backwards_compatibility_test.go
> <https://github.com/opencontainers/image-spec/pull/145#discussion_r67387069>
> :
>
> > +
> > +import (
> > +	"crypto/sha256"
> > +	"encoding/hex"
> > +	"fmt"
> > +	"strings"
> > +	"testing"
> > +
> > +	"github.com/opencontainers/image-spec/schema"
> > +)
> > +
> > +var compatMap = map[string]string{
> > +	"application/vnd.docker.distribution.manifest.list.v2+json": "application/vnd.oci.image.manifest.list.v1+json",
> > +	"application/vnd.docker.distribution.manifest.v2+json":      "application/vnd.oci.image.manifest.v1+json",
> > +	"application/vnd.docker.image.rootfs.diff.tar.gzip":         "application/vnd.oci.image.rootfs.tar.gzip",
> > +	"application/vnd.docker.container.image.v1+json":            "application/vnd.oci.image.serialization.config.v1+json",
>
> added this type on another PR
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/145/files/e94aa35aaa324ffc63d567ecb34d4914473702ed#r67387069>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACewN0gLt2OuGF2Nh0OFUuaYzQ4Hgn6Qks5qMYYfgaJpZM4I3Cyf>
> .
>